### ISSUE TITLE
Missing information in CSS chapter

### ISSUE THREAD

**Issue by edvin** [edvin/tornadofx-guide#12]
@t-boom did a quick sweep and notices some missing parts in the CSS chapter. We also need to figure out if the CSS information in the Wiki contains information that should be ported to the guide.

- Explanation for dimensions seems to ba lacking
- Explanation for `box` helper function seems to be lacking
- No explanation of functions like `removeClass`, `toggleClass`, `hasClass`, or `select` on `Node`s
- No explanation of more complex selectors (using infix functions and whatnot)
- No explanation of String literal selectors
- No reference to what classes and properties are available by default

**Comment by thomasnield** [comment]
I felt like I missed a few things on CSS. I can try to go through it later although it might be a week or two after I start my new job. You guys are welcome to improve it especially if you have a clear outline of what needs to be added. 

I have a feeling my involvement with the guide will be increasing again, because there are a lot of folks wanting to learn TornadoFX where I'm going.

**Comment by edvin** [comment]
That sounds perfect, Thomas! So happy for your involvement in this project, it means a lot.

**Comment by thomasnield** [comment]
@edvin absolutely, I am happy to. Almost done with Packt book as well. Ill have much more time when I hug the draft finished.

**Comment by t-boom** [comment]
In addition to more complex selectors, it's probably a good idea to show how to use selectors for things other than classes (i.e. elements, IDs, and pseudo-classes).

**Comment by thomasnield** [comment]
@t-boom You know this subject better than me, so feel free to make changes to the guide for this chapter if you like. I can look it over later.
